compiler/src/iree/compiler/Dialect/VM/IR/VMOps.cpp

```diff
@@ -201,6 +201,16 @@ void ExportOp::build(OpBuilder &builder, OperationState &result,
   result.attributes.append(attrs.begin(), attrs.end());
 }
 
+LogicalResult ExportOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
+  Operation *op = getOperation();
+  if (!symbolTable.lookupNearestSymbolFrom<IREE::VM::FuncOp>(
+          op, function_refAttr())) {
+    return op->emitError() << "vm.func op named '" << function_ref()
+                           << "' not found for export";
```

Comment:
Hmm... a few emitc test failures

https://source.cloud.google.com/results/invocations/108099f2-3a6a-49b6-bd28-4a1438424f65/targets;sort=name/iree%2Fgcp_ubuntu%2Fcmake%2Flinux%2Fx86-swiftshader%2Fpresubmit/log

```
FAILED: samples/static_library/simple_mul_c_module.h samples/static_library/simple_mul_c_module.o samples/static_library/simple_mul_emitc.h /tmpfs/src/github/iree/build/samples/static_library/simple_mul_c_module.h /tmpfs/src/github/iree/build/samples/static_library/simple_mul_c_module.o /tmpfs/src/github/iree/build/samples/static_library/simple_mul_emitc.h
cd /tmpfs/src/github/iree/build/samples/static_library && /tmpfs/src/github/iree/build/tools/iree-compile --iree-mlir-to-vm-c-module --iree-hal-target-backends=dylib-llvm-aot --iree-llvm-link-embedded=false --iree-llvm-link-static --iree-llvm-static-library-output-path=simple_mul_c_module.o /tmpfs/src/github/iree/samples/static_library/simple_mul.mlir -o simple_mul_emitc.h
/tmpfs/src/github/iree/samples/static_library/simple_mul.mlir:1:1: error: vm.func op named 'module_simple_mul_export_shim' not found for export
func.func @simple_mul(%arg0: tensor<4xf32>, %arg1: tensor<4xf32>) -> tensor<4xf32>
^
/tmpfs/src/github/iree/samples/static_library/simple_mul.mlir:1:1: note: see current operation: "vm.export"() {export_name = "simple_mul", function_ref = @module_simple_mul_export_shim, iree.abi.stub, ordinal = 0 : i32} : () -> ()
/tmpfs/src/github/iree/samples/static_library/simple_mul.mlir:0:0: error: failed during transform passes
```

Reply:
cc @marbre + @simon-camp 

https://github.com/iree-org/iree/pull/8057 added these "_export_shim" ops. I might want some help debugging why this new verifier is breaking under emitc.

Reply:
Perfect, thank you! Tests are passing now :)